sentinel/src/config.js

```diff
@@ -24,7 +24,7 @@ const loadTranslations = () => {
   return db.medic
     .query('medic-client/doc_by_type', options)
     .then(result => {
-      result.rows.forEach(row => (translations[row.doc.code] = row.doc.values));
+      result.rows.forEach(row => (translations[row.doc.code] = Object.assign(row.doc.custom, row.doc.default)));
```

Comment:
I think you need to make this same change in `api/src/config.js` for any translations we do in API.